src/components/AdminSettings/SIPBridge.vue

```diff
@@ -114,6 +131,19 @@ export default {
 			sipGroups: [],
 			dialInInfo: '',
 			sharedSecret: '',
+			dialOutEnabled: false,
+			currentSetup: {},
+			// FIXME replace with backend implementation
```

Comment:
Pending backend support

Reply:
@DorraJaouad can you help?
Basically we need a new store or something to bring the `data.features` from https://github.com/nextcloud/spreed/pull/10733/files#diff-b4c1956a877a395911806b992423657b94b60970a112e09d0d05ef01c6debe90R176 into this component here and when `dialout` is part of the list, we need to turn-off and disable the "SIP dialout" checkbox and show a warning: "The signaling server needs to be updated to enable this feature"